vite.config.ts

```diff
@@ -44,7 +44,7 @@ export default defineConfig(({ mode }) => {
     },
     server: commonEnv.API_PROXY
       ? {
-          port: 5172,
+          port: 5173,
```

Comment:
Presuming that we want to use the same port regardless of whether API_PROXY is defined.

Reply:
I thought I configured it the same, but I must have got the number wrong when I did it 😅